### ISSUE TITLE
[Patch] Subscribing to the Events stream in Open MCT causes real-time telemetry to drop in Open MCT-Yamcs 2.0.5

### ISSUE THREAD

**Issue by akhenry** [akhenry/openmct-yamcs#220]
#### Summary
A recent change breaks openmct-yamcs versions 2.0.5 - 2.1.0. Attempts to subscribe to the events stream will result in Yamcs dropping the WebSocket connection which will cause all realtime telemetry to drop out.

Navigating to a display that does not include events and refreshing the browser will restore real-time telemetry .

This issue was fixed in version 2.1.1 of the Open MCT-Yamcs plugin, however we need to patch the fix into a branch based on the 2.0.5 tag for deployment in a legacy production environment.

#### Impact Check List
- [X] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [X] Is there a workaround available?
- [X] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?

#### Steps to Reproduce
1. With Yamcs running and Open MCT connected, navigate to the Events stream
2. In the JavaScript console notice that the WebSocket is in an endless loop of connecting and disconnecting.

#### Environment
* Open MCT-Yamcs Version: 2.0.5 - 2.1.0
* Deployment Type: VIPER Prod
* Browser: Chrome 106.0.5249.119

**Comment by ozyx** [comment]
Small note, I don't think that the YAMCS API ever changed (there's nothing in their release notes about this breaking change). Instead, I think it may have been accidentally assumed that Events and Alarms would have the same subscription message structure. The offending lines [were added in 2.0.5](https://github.com/akhenry/openmct-yamcs/commit/8891603ef25f7ccc41786f3639fd7daa67930fcb) when we added Fault Management.

We should be able to catch regressions like these once we get e2e/CI running against openmct-yamcs.
